core-plugins/src/main/java/io/cdap/plugin/batch/aggregator/DedupConfig.java

```diff
@@ -52,6 +54,7 @@
     "of field and function is allowed. If this property is not set, one random record will be chosen from the " +
     "group of 'duplicate' records.")
   @Nullable
+  @Macro
```

Comment:
are nulls handled for both the macro properties?

Reply:
Yes